internal/benchmarks/test/fixtures/forc-projects/pyth-contract/src/main.sw

```diff
@@ -0,0 +1,934 @@
+contract;
```

Comment:
Based on a conversation at sync we probably don't want to maintain this large and complicated a dependency

@petertonysmith94 had recommended this [large contract](https://github.com/FuelLabs/fuels-ts/pull/3171/files#diff-1a0ad96ab237032f55549b8e2f5bab4e04d5235a3b134a794d7d17f5090d2f7b) do we agree?

Reply:
What was the rational behind not using the [large contract](https://github.com/FuelLabs/fuels-ts/blob/master/packages/fuel-gauge/test/fixtures/forc-projects/large-contract) with an ASM blob?

Reply:
I had wanted something that more closely resembles a realistic contract

Reply:
As we're only benchmarking the deployment of blobs, I would favour using the [large contract](https://github.com/FuelLabs/fuels-ts/blob/master/packages/fuel-gauge/test/fixtures/forc-projects/large-contract) with an ASM blob.

Reply:
ASM blob seems enough.

Reply:
aea62fb1f